MultiplayerMod/Game/Location/GameObjectLocation.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using MultiplayerMod.Game.Extension;
+using UnityEngine;
+
+namespace MultiplayerMod.Game.Location;
+
+[Serializable]
+public class GameObjectLocation {
```

Comment:
Would it make sense to make this surrogate? And pass game object in commands when required?

Reply:
You mean a surrogate for a `GameObject`? Not sure if it's useful for now and since I'm not sure I'll postpone it until real life scenarios kick in.